04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,75 @@
+const Product = require("../models/product");
+
+const getAllProductsStatic = async (req, res) => {
+  const products = await Product.find({ price: { $gt: 30 } })
+    .sort("price")
+    .select("name price");
+  res.status(200).json({ products, nbHits: products.length });
+};
+
+const getAllProducts = async (req, res) => {
+  const { featured, company, name, sort, fields, numericFilters } = req.query;
+  const queryObject = {};
+  if (featured) {
+    queryObject.featured = featured === "true" ? true : false;
+  }
+  if (company) {
+    queryObject.company = company;
+  }
+  if (name) {
+    queryObject.name = { $regex: name, $options: "i" };
+  }
+  if (numericFilters) {
+    const operatorMap = { //price>40, rating>=4
+      ">": "$gt",
```

Comment:
I got an items with price == 30 on a query for price > 30; possible bug?

<img width="1290" alt="Screen Shot 2022-08-28 at 11 17 31 AM" src="https://user-images.githubusercontent.com/58614260/187081438-3ad3a1cb-1586-44bd-8855-bbc7ad93c528.png">

Reply:
Update: I'm reviewing more CTD projects (I'm new to CTD) and seeing this behavior in other projects too, so I suspect this is more of a bug with the course than something you've done wrong here, per se.